src/pages/instances/forms/CreateImageFromInstanceForm.tsx

```diff
@@ -0,0 +1,131 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { useEventQueue } from "context/eventQueue";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { createImageAlias, createImageFromInstance } from "api/images";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { Link } from "react-router-dom";
+
+interface Props {
+  instance: LxdInstance;
+  close: () => void;
+}
+
+const CreateImageFromInstanceForm: FC<Props> = ({ instance, close }) => {
+  const eventQueue = useEventQueue();
+  const toastNotify = useToastNotification();
+
+  const notifySuccess = () => {
+    const published = (
+      <Link to={`/ui/project/${instance.project}/images`}>published</Link>
+    );
+    toastNotify.success(
+      <>
+        Image {published} from instance {instance.name}.
+      </>,
+    );
+  };
+
+  const formik = useFormik<{ alias: string; isPublic: boolean }>({
+    initialValues: {
+      alias: "",
```

Comment:
```suggestion
      alias: `from-instance-${instance.name}`,
```
I think we should use a default alias with the instance name here.